int32 robot_control_date
int32 robot_speed
